internal/services/contracts_events_consumer.go

```diff
@@ -689,6 +692,35 @@ func (c *ContractsEventsConsumer) dcnNewExpiration(e *ContractEventData) error {
 	return nil
 }
 
+// aftermarketDeviceAddressReset handles the event of the same name from the registry contract.
+func (c *ContractsEventsConsumer) aftermarketDeviceAddressReset(e *ContractEventData) error {
+	if e.ChainID != c.settings.DIMORegistryChainID || e.Contract != common.HexToAddress(c.settings.DIMORegistryAddr) {
```

Comment:
I noticed a comment elsewhere in this file that it would be great to remove this from each handler. Could we move it up to the case/ switch and group all events that are from a common contract together so we could just check it once?

Reply:
We should do that, but don't worry about it now.

Reply:
Let's add a test too.